treeherder/webapp/api/runnable_jobs.py

```diff
@@ -57,6 +78,31 @@ def list(self, request, project):
                 'state': 'runnable',
                 'result': 'runnable'})
 
+        for node in tc_graph['tasks']:
```

Comment:
Could you give a sample of what the tc_graph structure looks like? in principle this should be not hard since taskcluster is submitting this information to treeherder, but I guess we'll have to see...